plugins/src/main/java/arm/org/tron/plugins/ArchiveManifest.java

```diff
@@ -0,0 +1,25 @@
+package org.tron.plugins;
+
+import lombok.extern.slf4j.Slf4j;
+import org.tron.common.arch.Arch;
+
+/*
+  ARM platforms support RocksDB only; this command is not needed but retained for compatibility.
```

Comment:
--> "ARM platform only supports RocksDB, this command is not supported but retained for compatibility."